app/src/main/java/com/portfolio/tasky/usecases/PasswordPatternValidator.kt

```diff
@@ -0,0 +1,5 @@
+package com.portfolio.tasky.usecases
+
+interface PasswordPatternValidator {
```

Comment:
This, as well as the FullNameValidator, doesn't need to be an abstraction

Since they don't require any Android-specific dependencies

The only one that did, is the email one
That's why that one needs to have an abstraction